x-pack/plugins/ml/public/alerting/preview_alert_condition.tsx

```diff
@@ -0,0 +1,252 @@
+          defaultMessage: 'Job IDs:',
+        }),
+        description: jobIds.join(', '),
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.timeLabel', {
+          defaultMessage: 'Time: ',
+        }),
+        description: timestampIso8601,
+      },
+      {
+        title: i18n.translate('xpack.ml.previewAlert.scoreLabel', {
+          defaultMessage: 'Anomaly score:',
+        }),
+        description: score,
+      },
+      ...(topInfluencers && topInfluencers.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topInfluencersLabel', {
+                defaultMessage: 'Top influencers:',
+              }),
+              description: (
+                <ul>
+                  {topInfluencers.map((i) => (
+                    <li key={i.unique_key}>
+                      <EuiCode transparentBackground>{i.influencer_field_name}</EuiCode> ={' '}
+                      {i.influencer_field_value} [{i.score}]
+                    </li>
+                  ))}
+                </ul>
+              ),
+            },
+          ]
+        : []),
+      ...(topRecords && topRecords.length > 0
+        ? [
+            {
+              title: i18n.translate('xpack.ml.previewAlert.topRecordsLabel', {
+                defaultMessage: 'Top records:',
+              }),
+              description: (
+                <ul>
+                  {topRecords.map((i) => (
+                    <li key={i.unique_key}>
+                      <EuiCode transparentBackground>
+                        {i.function}({i.field_name})
+                      </EuiCode>{' '}
+                      {i.by_field_value} {i.over_field_value} {i.partition_field_value} [{i.score}]
+                    </li>
+                  ))}
+                </ul>
+              ),
+            },
+          ]
+        : []),
+    ];
+
+    return <EuiDescriptionList type={'row'} compressed={true} listItems={listItems} />;
+  }
+);
+export const PreviewAlertCondition: FC<PreviewAlertConditionProps> = ({
+  alertingApiService,
+  alertParams,
+}) => {
+  const [lookBehindInterval, setLookBehindInterval] = useState<string>();
+  const [areResultsVisible, setAreResultVisible] = useState<boolean>(true);
+  const [previewResponse, setPreviewResponse] = useState<PreviewResponse | undefined>();
+
+  const validators = useMemo(
+    () => composeValidators(requiredValidator(), timeIntervalInputValidator()),
+    []
+  );
+
+  const errors = useMemo(() => validators(lookBehindInterval), [lookBehindInterval]);
+
+  useEffect(
+    function resetPreview() {
+      setPreviewResponse(undefined);
+    },
+    [alertParams]
+  );
+
+  const testCondition = useCallback(async () => {
+    try {
+      const response = await alertingApiService.preview({
+        alertParams,
+        timeRange: lookBehindInterval!,
+      });
+      setPreviewResponse(response);
+    } catch (e) {
+      // TODO handle error
```

Comment:
is this a TODO for a follow up?

Reply:
Added in https://github.com/elastic/kibana/pull/89286/commits/c316b89a1c81fdcb582695e2af398a0c1b15a628